core/src/main/java/co/aikar/commands/BaseCommand.java

```diff
@@ -203,6 +341,7 @@ private void registerSubcommands() {
 
         for (Method method : this.getClass().getMethods()) {
             method.setAccessible(true);
+            if (method.isAnnotationPresent(Ignored.class)) { continue; }
```

Comment:
again this is not necessary. This can be done in the future with processors if someone wants.